api/src/pcapi/routes/public/serialization/venues.py

```diff
@@ -76,6 +76,7 @@ class OffererResponse(serialization.ConfiguredBaseModel):
     dateCreated: datetime.datetime = pydantic_v1.Field(..., alias="createdDatetime")
     name: str = pydantic_v1.Field(example="Structure A")
     siren: str | None = pydantic_v1.Field(example="123456789")
+    allowedOnAdage: bool
```

Comment:
Ça vaudrait le coup de rajouter une description du champ (rajouter un `ALLOWED_ON_ADAGE` dans `api/src/pcapi/routes/public/documentation_constants/fields.py`)
```suggestion
    allowedOnAdage: bool = fields.ALLOWED_ON_ADAGE 
```